master/internal/api_user.go

```diff
@@ -334,8 +344,10 @@ func (a *apiServer) PatchUser(
 	var ug *model.AgentUserGroup
 	if pug := req.User.AgentUserGroup; pug != nil {
 		ug = &model.AgentUserGroup{
-			UID: int(req.User.AgentUserGroup.AgentUid),
-			GID: int(req.User.AgentUserGroup.AgentGid),
+			UID:   int(req.User.AgentUserGroup.AgentUid),
+			GID:   int(req.User.AgentUserGroup.AgentGid),
+			User:  req.User.AgentUserGroup.AgentUser,
+			Group: req.User.AgentUserGroup.AgentGroup,
```

Comment:
is there any restriction  or can a user make an agent-user or agent-group association for anyone?

Reply:
:+1: